### PR TITLE
Adds a captain autodrobe

### PR DESCRIPTION
**Pull request by Ktlwjec**
So the locker isn't filled with five thousand outfits anymore (also fixed a duplicate camera on omega).

:cl:  
rscadd: Captain clothing items can now be found in the Captain's Autodrobe.
/:cl:

### CODE REVIEW

**Comment by fluffe9911** [comment]
hos version when also neat

**Comment by Szyszkrzyneczka** [comment]
can we uhh speedmerge this thing

File: code/modules/vending/autodrobe.dm [hunk 1]
```diff
@@ -225,3 +225,39 @@
 /obj/item/vending_refill/autodrobe
 	machine_name = "AutoDrobe"
 	icon_state = "refill_costume"
+
+/obj/machinery/vending/autodrobe/capdrobe
+	name = "\improper Captain's Autodrobe"
```
Comment (Alexandria404) [inline code/modules/vending/autodrobe.dm:230]:
```suggestion
	name = "\improper Condom's Dispenser"
```

**Review by Alexandria404 — CHANGES_REQUESTED** [review]

File: code/modules/vending/autodrobe.dm (hunk 1 above)
Comment (Szyszkrzyneczka) [inline code/modules/vending/autodrobe.dm]:
```suggestion
	name = "\improper Second Locker"
```

Comment (Szyszkrzyneczka) [reply code/modules/vending/autodrobe.dm]: oh wait
uhh wrong button

Comment (nesisknesis) [inline code/modules/vending/autodrobe.dm:230]:
```suggestion
	name = "\improper ComdomDrobe"
```

Comment (nesisknesis) [reply code/modules/vending/autodrobe.dm:230]: https://wiki.yogstation.net/wiki/Terminology#Comdom

Comment (Alexandria404) [reply code/modules/vending/autodrobe.dm:230]: As we all know, the cap is a condom. That is a well established fact. As such it is only natural that it is named in such a way that honors this long established ascertainment.
It is the Dispenser of the condom. The only logical conclusion being naming it the Condom`s dispenser.

Comment (nesisknesis) [reply code/modules/vending/autodrobe.dm:230]: Comdom*

**Pull request merged**